datasource/index.js

```diff
@@ -198,6 +198,7 @@ module.exports = yeoman.generators.Base.extend({
 
   installConnector: function() {
     var connector = this.availableConnectors[this.connector];
+    if (!connector) return;
```

Comment:
When we don't have any information about the custom connector selected for a new datasource, then I think we should assume the package name is same as the connector name and run `npm install {this.connector}`.

@amuramoto @superkhau @ritch @raymondfeng Thoughts?